renovate.json

```
@@ -1,5 +1,12 @@
 {
   "extends": [
     "config:base"
+  ],
+  "packageRules": [
+    {
+      "matchPaths": ["v3/"],
+      "matchPackageNames": ["unleash-server"],
+      "allowedVersions": "<=3.17.6"
```

Not that we're planning on it, but would it be better to do something like this in case we end up releasing another v3 patch?

```suggestion
      "allowedVersions": "< 4"
```

The docs seem to imply that that should work.